scripts/createDB.sql

```diff
@@ -0,0 +1,213 @@
+  `production_order_id` INT NOT NULL AUTO_INCREMENT,
+  `status` VARCHAR(45) NOT NULL,
+  `date` DATE NOT NULL,
+  `price` DECIMAL(10,2) NOT NULL,
+  PRIMARY KEY (`production_order_id`))
+  
+  collate = utf8mb4_unicode_ci;
+
+CREATE TABLE `soen_390_db`.`raw_material` (
+  `raw_material_id` INT NOT NULL AUTO_INCREMENT,
+  `raw_material_name` VARCHAR(45) NOT NULL,
+  `quantity` INT NOT NULL,
+  `price` DECIMAL(10,2) NOT NULL,
+  `vendor` VARCHAR(45),
+  PRIMARY KEY (`raw_material_id`),
+  UNIQUE INDEX `raw_material_name_UNIQUE` (`raw_material_name` ASC) VISIBLE)
+  
+  collate = utf8mb4_unicode_ci;
+    
+CREATE TABLE `soen_390_db`.`semi_finished_good` (
+  `semi_finished_good_id` INT NOT NULL AUTO_INCREMENT,
+  `semi_finished_good_name` VARCHAR(45) NOT NULL,
+  `quantity` INT NOT NULL,
+  `price` DECIMAL(10,2) NOT NULL,
+  `vendor` VARCHAR(45),
+  PRIMARY KEY (`semi_finished_good_id`),
+  UNIQUE INDEX `semi_finished_good_name_UNIQUE` (`semi_finished_good_name` ASC) VISIBLE)
+  
+  collate = utf8mb4_unicode_ci;
+    
+CREATE TABLE `soen_390_db`.`finished_good` (
+  `finished_good_id` INT NOT NULL AUTO_INCREMENT,
+  `finished_good_name` VARCHAR(45) NOT NULL,
+  `quantity` INT NOT NULL,
+  `price_of_construction` DECIMAL(10,2) NOT NULL,
+  `price_of_selling` DECIMAL(10,2) NOT NULL,
+  PRIMARY KEY (`finished_good_id`),
+  UNIQUE INDEX `finished_good_name_UNIQUE` (`finished_good_name` ASC) VISIBLE)
+  
+  collate = utf8mb4_unicode_ci;
+  
+CREATE TABLE `soen_390_db`.`ordered_raw_material` (
+  `production_order_id` INT NOT NULL,
+  `raw_material_id` INT NOT NULL,
+  `quantity` INT NOT NULL,
+  `price` DECIMAL(10,2) NOT NULL,
+  PRIMARY KEY (`production_order_id`, `raw_material_id`),
+  CONSTRAINT `rawMaterialProductionOrderIDForeignKey`
+    FOREIGN KEY (`production_order_id`)
+    REFERENCES `soen_390_db`.`production_order` (`production_order_id`)
+    ON DELETE CASCADE
+    ON UPDATE NO ACTION,
+  CONSTRAINT `rawMaterialIDForeignKey`
+    FOREIGN KEY (`raw_material_id`)
+    REFERENCES `soen_390_db`.`raw_material` (`raw_material_id`)
+    ON DELETE NO ACTION
+    ON UPDATE CASCADE)
+    
+	collate = utf8mb4_unicode_ci;
+    
+CREATE TABLE `soen_390_db`.`ordered_semi_finished` (
+  `production_order_id` INT NOT NULL,
+  `semi_finished_good_id` INT NOT NULL,
+  `quantity` INT NOT NULL,
+  `price` DECIMAL(10,2) NOT NULL,
+  PRIMARY KEY (`production_order_id`, `semi_finished_good_id`),
+  CONSTRAINT `semiFinishedProductionOrderIDForeignKey`
+    FOREIGN KEY (`production_order_id`)
+    REFERENCES `soen_390_db`.`production_order` (`production_order_id`)
+    ON DELETE CASCADE
+    ON UPDATE NO ACTION,
+  CONSTRAINT `semiFinishedGoodIDForeignKey`
+    FOREIGN KEY (`semi_finished_good_id`)
+    REFERENCES `soen_390_db`.`semi_finished_good` (`semi_finished_good_id`)
+    ON DELETE NO ACTION
+    ON UPDATE CASCADE)
+    
+	collate = utf8mb4_unicode_ci;
+
+CREATE TABLE `soen_390_db`.`ordered_finished` (
+  `customer_order_id` INT NOT NULL,
```

Comment:
```suggestion
  `production_order_id` INT NOT NULL,
```

This is for the construction order.

We should have another table for the relation customer order and finished good